metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -42,7 +42,8 @@ case class UserConfiguration(
     enableStripMarginOnTypeFormatting: Boolean = true,
     enableIndentOnPaste: Boolean = false,
     excludedPackages: Option[List[String]] = None,
-    fallbackScalaVersion: Option[String] = None
+    fallbackScalaVersion: Option[String] = None,
+    disableTestCodeLenses: Boolean = false
```

Comment:
Can we just have it as an enum
```suggestion
    testUserInterface: TestUserInterfaceType = TestUserInterfaceType.Lenses
```
?

`disableTestCodeLenses` doesn't really correspond to what is show on UI.